### ISSUE TITLE
Work profile?

### ISSUE THREAD

**Issue by JonnyTech** [phhusson/treble_experimentations#554]
Does the AOSP9 GSI support the Android `work profile`? Trying to install `Shelter` from F-droid but states that it is incompatible because it requires `android.software.managed_users`. Is this a limitation of the ROM or my device (Redmi GO)?

![shelter](https://user-images.githubusercontent.com/3584732/59151349-4ce6d800-8a29-11e9-890a-b4ae9028f03d.jpg)

**Comment by phhusson** [comment]
It's a limitation of your device, devices declared low_ram have disabled managed_users.

You can try to edit your /vendor/build.prop to remove ro.config.low_ram=true line and reboot, but Android will eat much more RAM, so you might want to revert later.

**Comment by phhusson** [comment]
(FWIW I tried it on my Mi Mix 3, it worked. Though non-managed apps couldn't access to webview anymore. I'll have to investigate this)

**Comment by JonnyTech** [comment]
Great, thanks @phhusson. When I get some time then I shall try it. What do you mean by "non managed"? Is there anything else that the low_ram flag affects?

**Comment by phhusson** [comment]
When you have a work profile, you have "managed apps" on one side, and previously installed apps are "unmanaged apps"

When I enable work profile, the apps previously installed, so outside of work profile, can no longer connect with webview (which should be available both inside the work profile and outside)

**Comment by JonnyTech** [comment]
Yay, it works, thanks @phhusson.

A side effect is that I now have a secondary navigation / gesture bar at the botton of my screen above my phones standard navigation bar. This means that I have less screen estate. I already had Pie gestures enabled before this change without the additional bar. Is it possible to disable it? Gestures worked perfectly without it and I do not need a permanent "pill" at the bottom of each screen

**Comment by phhusson** [comment]
You mean you have a software navbar in addition to hardware navbar?

**Comment by JonnyTech** [comment]
Yes, indeed, that is the correct terminology. Only with `low_ram=false`.

**Comment by JonnyTech** [comment]
Any update on how to disable the software navbar? Have tried both the vendor build prop and the phh treble settings. Still there with v221.

![img_5](https://user-images.githubusercontent.com/3584732/87660133-79097800-c756-11ea-8219-09166b7bc158.jpg)
